ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -41,7 +41,7 @@
                 let body = action.payload?.body;
                 if (body) {
                     if (body.redirectUrl) {
-                        let redirectURL = self._path + '.guideThankYouPage.html';  //default ThankYouPage Path
+                        let redirectURL = body.redirectUrl;  //default ThankYouPage Path
```

Comment:
This is fixed by @sakshi-arora1

Reply:
Fix is not yet present in dev or master

Reply:
The current test failure is because of this change, until af2-rest-api changes are merged, test would fail.

Reply:
body.redirectUrl = '/content/forms/af/testsigneraf2/jcr:content/guideContainer.guideThankYouPage.html'
self._path = '/content/forms/af/testsigneraf2/jcr:content/guideContainer'

So ideally nothing has changed in logic.

cc : @sakshi-arora1 @rismehta